.eslintrc

```diff
@@ -19,6 +19,7 @@
   "rules": {
     "no-underscore-dangle": "off",
     "no-param-reassign": "off",
-    "import/no-extraneous-dependencies": "off"
+    "import/no-extraneous-dependencies": "off",
+    "jsx-a11y/no-static-element-interactions": "off"
```

Comment:
This should not be disable globally here but locally in the file (even more accurately - few lines in the file should be disabled).